src/libecalc/core/ecalc.py

```diff
@@ -12,20 +12,20 @@
 from libecalc.core.models.fuel import FuelModel
 from libecalc.core.result import EcalcModelResult
 from libecalc.core.result.emission import EmissionResult
-from libecalc.dto.graph import Graph
+from libecalc.dto.component_graph import ComponentGraph
 from libecalc.dto.types import ConsumptionType
 
 
 class EnergyCalculator:
     def __init__(
         self,
-        graph: Graph,
+        graph: ComponentGraph,
```

Comment:
EnergyEquipmentNetwork? @olelod what is a good name to name the overall network (?) of energy equipment in an installation?

Reply:
Process Flow Diagram perhaps

Reply:
Can we also come up with a name for the network where the edges are streams?